src/dropdown/dropdown.js

```diff
@@ -170,9 +162,50 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
         return;
       }
       dropdownCtrl.dropdownMenu = element;
+
+      var dropdownMenu = element,
+          dropdown = element.parent(),
```

Comment:
@wesleycho Currently, CSS code defines that `.dropdown-menu` **must be** a direct child of `dropdown` element to be displayed:
```
.open > .dropdown-menu {
  display: block;
}
```